net/netstinky/files/nsids.init

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+START=95
+STOP=01
+
+CONFIGURATION=netstinky
+
+validate_ids_section()
+{
+    uci_load_validate netstinky netstinky "$1" "$2" \
+            'interface:string:lan' \
+            'port:uinteger:8089' \
+            'updatehost:string' \
+            'updateport:uinteger' \
+            'sslnoverify:bool:false'
+}
+
+netstinky_instance()
+{
+    [ "$2" = 0 ] || {
+        echo "validation failed"
+        return 1
+    }
+
+    procd_open_instance
+    procd_set_param command /usr/bin/nsids
+
+    if [ -n "${interface}" ]; then
+        local iface
+        network_get_device iface "${interface}"
+        procd_append_param command -i "${iface}"
+    fi
+    procd_append_param command -p "${port}"
+    [ -n "${updatehost}" ] && procd_append_param command --update-host "${updatehost}"
+    [ -n "${updateport}" ] && procd_append_param command --update-port "${updateport}"
+    [ "${sslnoverify}" -eq 1 ] && procd_append_param command --ssl-no-verify
+
+    procd_set_param stderr 1
+    procd_set_param stdout 1
+
+    procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
```

Comment:
if those are the defaults, just leave them off, otherwise it's just cargo culting that will be harder to determine the true cause for down the road.

Reply:
I've removed line 43 as it was in fact the defaults.

The `nsids` program logs to stdout and stderr (depending on severity) as it is intended to run on a few different platforms and doing it that way prevents us from having to depend on a particular logging solution.

As I understand it, procd expects a service to be running in the foreground anyway, so logging via stdout/stderr seemed appropriate.